lit_gpt/lora.py

```diff
@@ -390,11 +456,51 @@ class Config(BaseConfig):
             "This scaling helps to reduce the need to retune hyperparameters when we vary r"
             https://arxiv.org/pdf/2106.09685.pdf (section 4.1)
         dropout: dropout that is applied on the input in the LoRA branch (before multiplying by matrix A)
+        to_*: either apply LoRA to the specified weights or not
     """
 
     r: int = 0.0
     alpha: int = 1.0
     dropout: float = 0.0
+    to_query: bool = False
+    to_key: bool = False
+    to_value: bool = False
+    to_projection: bool = False
+    to_mlp: bool = False
+    to_head: bool = False
+
+    @property
+    def mlp_class(self) -> Type:
+        # `self._mlp_class` cannot be the type to keep the config json serializable
+        obj = lit_gpt.lora if self.to_mlp else lit_gpt.model
+        return getattr(obj, self._mlp_class)
+
+class GptNeoxMLP(nn.Module):
```

Comment:
Can you put the MLP's below the GPT model? This is how it's ordered in `model.py`

Reply:
Done